compiler/src/iree/compiler/Utils/Indexing.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Utils/Indexing.h"
+
+using namespace mlir;
+using namespace mlir::iree_compiler;
+
```

Comment:
I think the style convention is to use `namespace mlir::iree_compiler {` instead of prefixing function definitions

Reply:
I feel like a lot of the upstream code I've seen uses prefixing - is IREE different?

Reply:
Yeah IREE differs from upstream in a few ways (I don't know them all though). For example, we prefer braces after if/for/while when the line spans multiple lines.

Here's Utils.cpp for reference: https://github.com/iree-org/iree/blob/7e6c5ec26332c55d1912d3f7fd2cf2350a936768/compiler/src/iree/compiler/Codegen/Utils/Utils.cpp#L43